apps/src/templates/sectionProgress/progressTables/ProgressTableContainer.jsx

```diff
@@ -204,52 +255,101 @@ class ProgressTableContainer extends React.Component {
     };
   };
 
+  detailContentViewProps() {
+    return {
+      lessonCellFormatters: this.detailCellFormatters,
+      extraHeaderFormatters: [
+        getLevelIconHeaderFormatter(this.props.scriptData)
+      ],
+      includeHeaderArrows: true
+    };
+  }
+
+  summaryContentViewProps() {
+    return {
+      columnWidths: new Array(this.props.scriptData.stages.length).fill(
+        SUMMARY_VIEW_COLUMN_WIDTH
+      ),
+      lessonCellFormatters: this.summaryCellFormatters,
+      includeHeaderArrows: false
+    };
+  }
+
   render() {
+    const isDetailView = this.props.currentView === ViewType.DETAIL;
+
+    const studentListHeaders = [i18n.lesson()];
+    isDetailView && studentListHeaders.push(i18n.levelType());
+
+    const contentViewProps = isDetailView
+      ? this.detailContentViewProps()
+      : this.summaryContentViewProps();
+
     return (
-      <div style={styles.container} className="progress-table">
-        <div style={styles.studentList} className="student-list">
-          <ProgressTableStudentList
-            ref={r => (this.studentList = r)}
-            headers={[i18n.lesson(), ...(this.props.extraHeaderLabels || [])]}
-            rows={this.state.rows}
-            onRow={this.onRow}
-            sectionId={this.props.section.id}
-            scriptData={this.props.scriptData}
-            studentTimestamps={this.props.studentTimestamps}
-            localeCode={this.props.localeCode}
-            onToggleRow={this.onToggleRow}
-          />
-        </div>
-        <div style={styles.contentView} className="content-view">
-          <ProgressTableContentView
-            ref={r => (this.contentView = r)}
-            rows={this.state.rows}
-            onRow={this.onRow}
-            needsGutter={this.needsContentHeaderGutter()}
-            onScroll={this.onScroll}
-            scriptData={this.props.scriptData}
-            lessonOfInterest={this.props.lessonOfInterest}
-            onClickLesson={this.props.onClickLesson}
-            columnWidths={this.props.columnWidths}
-            lessonCellFormatters={this.props.lessonCellFormatters}
-            extraHeaderFormatters={this.props.extraHeaderFormatters}
-            includeHeaderArrows={this.props.includeHeaderArrows}
-          />
+      <div>
```

Comment:
NIT: a comment that this is the container for the table+legend would be helpful. not worth adding unless making other changes though